applications/API/serializers.py

```
@@ -0,0 +1,487 @@
+    Serializer for InsultCategory model.
+
+    Handles the listing and retrieval of insult categories.
+
+    Attributes:
+        category_key (str): Unique identifier for the category
+        name (str): Display name of the category
+    """
+
+    class Meta:
+        model = InsultCategory
+        fields = ["category_key", "name"]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)